pkg/pillar/cmd/volumemgr/initialvolumestatus.go

```diff
@@ -86,8 +76,9 @@ func populateInitialVolumeStatus(ctx *volumemgrContext, dirName string) {
 		volumeUUID, err := uuid.FromString(volumeID)
 		if err != nil {
 			log.Errorf("populateInitialVolumeStatus: Invalid volume UUIDStr(%s) in "+
-				"filename (%s). err: %s. XXX ignoring the File",
+				"filename (%s). err: %s. Deleting the File",
 				volumeID, filelocation, err)
+			deleteFile(filelocation)
```

Comment:
This is still doing to mess up the old files, no? Or is the assumption that this delete only happens in /persist/vault/volumes?

Reply:
No, delete will only happen in /persist/vault/volumes

Reply:
OK